app/src/main/java/com/github/se/signify/ui/screens/challenge/ChronoChallengeGameScreen.kt

```diff
@@ -40,7 +59,7 @@ fun ChronoChallengeGameScreen(
     navigationActions: NavigationActions,
     userSession: UserSession,
     challengeRepository: ChallengeRepository,
-    handLandMarkViewModel: HandLandMarkViewModel,
+    handLandMarkViewModel: HandLandmarkViewModel,
```

Comment:
```suggestion
    handLandmarkViewModel: HandLandmarkViewModel,
```
Same as before, you can do the changes

Reply:
Done.